### PR TITLE
[EPPETA-69] Remove compile and runtime warnings.

### PR DESCRIPTION
**Pull request by jleiva-gap**
- Update .NET projects to remove warnings
- Run VS Clean Up.
- Upgrade nuget System.IdentityModel.Tokens.Jwt to fix vulnerability.
- Update React project to fix warnings.

### CODE REVIEW

File: src/webapi/Controllers/CandidateController.cs [hunk 1]
```diff
@@ -24,17 +24,14 @@ public CandidateController(IODSAPIAuthenticationConfigurationService service, IE
         public async Task<ActionResult<IEnumerable<Candidate>>> GetCandidates()
         {
             var candidates = await _evaluationRepository.GetAllCandidates();
-            if (candidates != null)
-            {
-                return Ok(candidates.Select(c => new Candidate()
+            return candidates != null
```
Comment (stevenarnold-ed-fi) [inline src/webapi/Controllers/CandidateController.cs]: @jleiva-gap The ternary and casting here make this difficult to read, suggest:

`
if (candidates is null) { return Ok(new List<Candidate>()); }
`

`
            return Ok(candidates.Select(c => new Candidate()
            {
                CandidateName = $"{c.FirstName} {c.LastName}",
                PersonId = c.PersonId,
                SourceSystemDescriptor = c.SourceSystemDescriptor
            }).ToList());
`

File: src/webapi/Evaluations/Models/Candidate.cs [hunk 2]
```diff
@@ -30,32 +30,36 @@ public class Candidate
...
-                firstName: candidate.FirstName,
-                lastSurname: candidate.LastName,
-                personReference: new EdFiPersonReference
-                {
-                    PersonId = candidate.PersonId,
-                    SourceSystemDescriptor = candidate.SourceSystemDescriptor
-                }
-            );
+        {
+            return new TpdmCandidate
```
Comment (stevenarnold-ed-fi) [inline src/webapi/Evaluations/Models/Candidate.cs:53]: Curious on the change here, was it a personal preference or a tool suggesting the change? Either way is good, no update needed

**Review by stevenarnold-ed-fi — CHANGES_REQUESTED** [review]

**Comment by stevenarnold-ed-fi** [comment]
@jleiva-gap I attempted to load an EvaluationElementRating through the ODS/API swagger and was getting a 500 error, so I'm going to make the assumption that the error we're receiving on the elements API call we discussed in slack is an issue with that specific ODS/API and not an error in this code. If there is a bug in the elements code from these changes we'll fix it later.

Only the one change requested in the Candidates Controller and then we can merge.

File: src/webapi/Evaluations/Models/Candidate.cs (hunk 2 above)
Comment (jleiva-gap) [reply src/webapi/Evaluations/Models/Candidate.cs:53]: It was changed by the Visual Studio Clean Up tool.

**Review by stevenarnold-ed-fi — APPROVED** [review]

**Pull request merged**